EnhancedMediaPlayer/Util/Util.Date.swift

```diff
@@ -0,0 +1,27 @@
+//  DateUtils.swift
+
+import Foundation
+
+extension Util {
+    class Date {
+        static func timeFormat(_ timeInSeconds: Float64, totalTimeInSeconds: Float64) -> String {
+            let formatter = DateComponentsFormatter()
+
+            if totalTimeInSeconds >= Constants.secondsPerHour {
+                formatter.allowedUnits = [.hour, .minute, .second]
+            } else {
+                formatter.allowedUnits = [.minute, .second]
+            }
+            formatter.unitsStyle = .positional
+            formatter.zeroFormattingBehavior = [.pad]
+
+            return formatter.string(from: TimeInterval(timeInSeconds))!
+        }
+    }
+}
+
+extension Util.Date {
+    enum Constants {
+        static let secondsPerHour: Float64 = 3600
```

Comment:
Again, why not use `TimeInterval` here? It's the standard way to denote seconds in `Swift`